credoai/lens/pipeline_creator.py

```diff
@@ -38,7 +39,14 @@ def process_evidence_requirements(evidence_requirements: List[EvidenceRequiremen
                 metrics.add(labels["metric_type"])
             elif "metric_types" in labels:
                 metrics = metrics.union(labels["metric_types"])
+            elif "table_name" in labels:
```

Comment:
I don't think this work at all.

Reply:
Gonna try and fix. This breaks if you ask for `disaggregated_table` and also doesn't work for performance, which doesn't return an `roc_curve` table_name